actions/templates.go

```diff
@@ -264,19 +263,21 @@ func DeleteTemplate(c *gin.Context) {
 		return
 	}
 
-	template, err := storage.GetTemplate(c, id, u.ID)
-	if err != nil {
-		c.JSON(http.StatusUnprocessableEntity, gin.H{
-			"message": "Template not found",
-		})
-		return
-	}
-
+	u := middleware.GetUser(c)
 	service := templatesvc.NewTemplateService(storage.GetFromContext(c), s3.GetFromContext(c))
 
-	err = service.DeleteTemplate(c, template)
+	err = service.DeleteTemplate(c, id, u.ID)
 	if err != nil {
-		logger.From(c).WithField("template", *template).WithError(err).Error("Unable to delete template.")
+		if errors.Is(err, gorm.ErrRecordNotFound) {
```

Comment:
If a template is not found, we return ok response here. See the other delete endpoints for reference.

Reply:
Since we dont fetch it before deleting this error wont happen